### PR TITLE
AMKO readme changes

### PR DESCRIPTION
**Pull request by swathinsankaran**
This PR updates the version information and dependencies present in the readme.

### CODE REVIEW

**Comment by swathinsankaran** [comment]
Hi @sudswasavi, @monotosh-avi, @akshayhavile, @chauhanshubham,
Could you please review this PR?

**Comment by swathinsankaran** [comment]
build amko

File: README.md [hunk 1]
```diff
@@ -21,14 +21,14 @@ For Kubernetes clusters:
 | **Components** | **Version** |
 | -------------- | ----------- |
 | Kubernetes     | 1.16+       |
```
Comment (monotosh-avi) [inline README.md:23]: I think we would have to update kubernetes and openshift versions too based AKO requirements.

Comment (swathinsankaran) [reply README.md]: I have updated the versions based on the documents https://github.com/vmware/load-balancer-and-ingress-services-for-kubernetes/blob/release-1.6.1/buildsettings.json and https://avinetworks.com/docs/ako/1.6/ako-compatibility-guide.

**Review by monotosh-avi — APPROVED** [review]

**Review by RiyaGaniga-avi — APPROVED** [review]

**Pull request merged**